resolution: 0.08
origin_x: -36
origin_y: -36
size_x: 900
size_y: 900